src/storage/manager.rs

```diff
@@ -17,11 +17,21 @@ use crate::{
 /// The storage used by the manager.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub(crate) enum ManagerStorage {
-    #[cfg(feature = "stronghold")]
     /// Stronghold storage.
+    #[cfg(feature = "stronghold")]
     Stronghold,
     /// RocksDB storage.
+    #[cfg(feature = "rocksdb")]
     Rocksdb,
+    /// Storage backed by a Map in memory, for testing
```

Comment:
```suggestion
    /// Storage backed by a Map in memory
```
Removing because it may not only be for testing, browser may use that in the future.